terraform/open.fec.gov.tf

```diff
@@ -6,16 +6,10 @@ resource "aws_route53_zone" "open_fec_gov_zone" {
   }
 }
 
-resource "aws_route53_record" "open_fec_gov_open_fec_gov_a" {
-  zone_id = "${aws_route53_zone.open_fec_gov_zone.zone_id}"
-  name    = "open.fec.gov."
+resource "aws_route53_record" "open_fec_gov_open_fec_gov_a" {	
```

Comment:
Why are we keeping this record and not clearing it out?

Reply:
FEC expressed some concerns about removing it.  But I think we can probably remove it.  Good catch.